app/src/main/java/com/example/weatherapp/models/LocationRepository.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.weatherapp.models
+
+import retrofit2.Retrofit
+
+class LocationRepository {
```

Comment:
I liked that you created two repositories. One for Location and other for WeatherDetails 👏

Reply:
Thanks very much Phellipe